### DISCUSSION TITLE
Possible to include embedded .pdfs in note context?

### DISCUSSION THREAD

**Discussion by fneuk** [logancyang/obsidian-copilot#49] [Ideas]
Hey Logan,

thanks for you work, really appreciate it. 

Would it be possible that the CoPilot also recognizes .pdfs as note context for a QA? or .pdfs embedded in a note?

Kind regards

**Comment by logancyang** [comment]
Great suggestion! Adding this to the roadmap.

**Comment by LazyAIEnjoyer** [comment]
Yeah it would be very cool to be able to summarize, make automatic notes or ask questions about books .epubs or other .pdfs inside your vault. Kinda the same as using GPT-4 with web search functionality inside obsidian, but then with your local digital books or research papers that you have stored in the vault.
